### PR TITLE
Add InstantGeo Detection

### PR DESCRIPTION
**Pull request by azeemhassni**
This PR adds instantgeo.info to the list of technologies, this is my first contribution to this repo I'm happy to hear any feedback if I'm doing anything wrong. :)

### CODE REVIEW

File: src/technologies/i.json [hunk 1]
```diff
@@ -1164,6 +1164,24 @@
     "saas": true,
     "website": "https://www.insided.com"
   },
+  "InstantGeo": {
+    "cats": [59],
+    "description": "InstantGeo is a service that provides IP Geolocation to web pages",
+    "icon": "instantgeo.svg",
+    "implies": [],
```
Comment (nurbek91) [inline src/technologies/i.json:1171]: "implies": [], 
Can be skipped if there is nothing to write there

File: src/technologies/i.json [hunk 2]
```diff
@@ -1164,6 +1164,24 @@
     "saas": true,
     "website": "https://www.insided.com"
   },
+  "InstantGeo": {
+    "cats": [59],
+    "description": "InstantGeo is a service that provides IP Geolocation to web pages",
```
Comment (nurbek91) [inline src/technologies/i.json:1169]: "description": "InstantGeo is a service that provides IP geolocation to web pages.",

File: src/technologies/i.json [hunk 3]
```diff
@@ -1164,6 +1164,24 @@
...
+    "icon": "instantgeo.svg",
```
Comment (nurbek91) [inline src/technologies/i.json:1170]: "icon": "instantgeo.svg", - >  "icon": "InstantGeo.svg",

File: src/technologies/i.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/i.json:1169]: [InstantGeo.svg.zip](https://github.com/wappalyzer/wappalyzer/files/9105772/InstantGeo.svg.zip)
32x32

**Review by nurbek91 — CHANGES_REQUESTED** [review]

**Comment by nurbek91** [comment]
Hey mate, thanks for PR!!
Everything looks great. Changes are required simply to the style. 
It is also one of the requirements to attach examples(3-4 links if possible)

File: src/technologies/i.json (hunk 1 above)
Comment (azeemhassni) [reply src/technologies/i.json]: Removed it in my recent commit, thanks for pointing it out.

File: src/technologies/i.json (hunk 2 above)
Comment (azeemhassni) [reply src/technologies/i.json]: Updated and the description should be as suggested now.

File: src/technologies/i.json (hunk 3 above)
Comment (azeemhassni) [reply src/technologies/i.json]: Makes sense, so the icon name should exactly match the technology/app name. I have updated it and made sure the SVG rendering size is 32x32 px.

**Review by nurbek91 — APPROVED** [review]
Thanks mate!!

**Pull request merged**